apps/web-giddh/src/app/settings/permissions/form/form.component.ts

```
@@ -143,11 +143,14 @@ export class SettingPermissionFormComponent implements OnInit, OnDestroy {
             } else {
                 this.isSuperAdminCompany = false;
             }
-            if (activeCompany?.moduleRestrictionStatus) {
+            if (activeCompany?.subscription?.planDetails?.restrictedModules.hasOwnProperty(this.restrictedModules.Users) && activeCompany?.moduleRestrictionStatus) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)